lib/api/controllers/collectionController.js

```diff
@@ -220,21 +233,20 @@ class CollectionController {
       .map(collection => collection.name);
 
     if (type === 'realtime') {
-      let realtimeResponse = {type, collections: {realtime: realtimeCollections}};
+      const realtimeResponse = {type, collections: {realtime: realtimeCollections}};
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 84 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)